internal/blueprintload/blueprintload.go

```diff
@@ -0,0 +1,67 @@
+package blueprintload
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+
+	"github.com/BurntSushi/toml"
+
+	"github.com/osbuild/images/pkg/blueprint"
```

Comment:
I guess that the ship has already sailed in BIB world, but the initial intention was for `images/pkg/blueprint` not to be used to decode user-provided blueprints (as it's the case in osbuild-composer).

Reply:
Yeah, that ship has sailed unfortunately /o\. Sorry, I just wasn't aware of this.